### PR TITLE
Changed example in Surface.set_fill_by_value()

### PR DESCRIPTION
**Pull request by Leo-Torch**
## Overview: What does this pull request change?

This changes the example in the set_fill_by_value() method.

## Motivation and Explanation: Why and how do your changes improve the library?

This change makes it easier for new users to understand how to use the method.
## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

**Comment by behackl** [comment]
Hello! Thank you for your contribution!

Can you please explain, in how far your changes make the method easier to use? I somehow prefer the look and feel of the current example.

**Comment by Leo-Torch** [comment]
> Hello! Thank you for your contribution!
> 
> Can you please explain, in how far your changes make the method easier to use? I somehow prefer the look and feel of the current example.

In the original example, I didn't feel that it effectively illustrated the `FillByValue` gradient very clearly to people who are planning the use the method. Only a small streak of yellow can be seen on the graph with a large portion of the coloring being green. By making the gradient more varied it becomes clear how you can use the method, and what you can expect to see from it.

**Comment by behackl** [comment]
So, here is the original version: <https://docs.manim.community/en/stable/reference/manim.mobject.three_dimensions.Surface.html>, and here is the version from your PR: <https://manimce--2431.org.readthedocs.build/en/2431/reference/manim.mobject.three_dimensions.Surface.html>

The aim of the original example is to have a gradient along the z-axis, with large function values being green and small ones being red. Semantically, I slightly prefer that over setting the gradient along the y-axis. But I definitely agree with you: the example could be much better! Do you think changing the camera orientation to make the valleys and mountains of the function more visible would help?

**Comment by Leo-Torch** [comment]
I definitely agree. I'll make a new commit with a better camera orientation for the z-axis shortly.

File: manim/mobject/three_dimensions.py [hunk 1]
```diff
@@ -210,7 +210,7 @@ def param_surface(u, v):
                         u_range=[0, 5],
                         )
                     surface_plane.set_style(fill_opacity=1)
-                    surface_plane.set_fill_by_value(axes=axes, colors=[(RED, -0.4), (YELLOW, 0), (GREEN, 0.4)], axis = 1)
+                    surface_plane.set_fill_by_value(axes=axes, colors=[(RED, -0.5), (YELLOW, 0), (GREEN, 0.5)], axis = 2)
```
Comment (behackl) [inline manim/mobject/three_dimensions.py:213]: Minor formatting change.

```suggestion
                    surface_plane.set_fill_by_value(axes=axes, colors=[(RED, -0.5), (YELLOW, 0), (GREEN, 0.5)], axis=2)
```

**Review by behackl — CHANGES_REQUESTED** [review]
Looks good to me, thank you! I have one minor suggestion with respect to formatting, but otherwise this can be merged. Please leave a comment or request a review when you have applied the changes!

**Review by Darylgolden — APPROVED** [review]

**Pull request merged**